### PR TITLE
Add translation of the messages to brazilian portuguese

### PR DESCRIPTION
**Pull request by EduardoJM**
This only add translation messages to brazilian portuguese.

### CODE REVIEW

**Review by auvipy — CHANGES_REQUESTED** [review]
can you please mention some fellow brazilian portugues speakers to recheck this?

**Comment by EduardoJM** [comment]
> can you please mention some fellow brazilian portugues speakers to recheck this?

thats a good idea. i will made it.

File: django_celery_results/locale/pt_BR/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+msgid "Current state of the task being run"
+msgstr "Status atual da tarefa em execução"
+
+#: models.py:55
+msgid "Worker"
+msgstr "Worker"
+
+#: models.py:55
+msgid "Worker that executes the task"
+msgstr "Worker que executou a tarefa"
```
Comment (Fabio3rs) [inline django_celery_results/locale/pt_BR/LC_MESSAGES/django.po:87]: Acredito que aqui deveria ser "Woker que executa a tarefa" ou "Worker que roda a tarefa"
Considerando que "executes" parece ser tempo presente

I think that here should be "Woker que executa a tarefa" or "Worker que roda a tarefa", seeing that "executes" is present time.

**Review by Fabio3rs — CHANGES_REQUESTED** [review]

**Comment by EduardoJM** [comment]
Reviewed by @Fabio3rs and by another people (who speaks in english and in brazilian portuguese, but no have github account 5618bc3).

**Comment by auvipy** [comment]
OK thanks

**Review by auvipy — APPROVED** [review]

**Comment by auvipy** [comment]
Thank you. I also have started  learning Brazilian Portuguese in duolingo & European Portuguese in youtube. though I don't understand it quite well as of yet.

**Pull request merged**